scripts/joint_trajectory_file_playback.py

```diff
@@ -233,12 +239,38 @@ def find_start_offset(pos):
             cur_cmd = [cmd['right_gripper']]
             self._add_point(cur_cmd, 'right_gripper', values[0] + start_offset)
 
+    def _feedback(self, data):
+        if not self._get_trajectory_flag():
+            self._set_trajectory_flag(value=True)
+
+    def _set_trajectory_flag(self, value=False):
+        # Lock
+        self._lock.acquire()
+        # Assign a value to the flag
+        self._arm_trajectory_started = value
+        # Unlock
+        self._lock.release()
+
+    def _get_trajectory_flag(self):
+        temp_flag = False
+        # Lock
+        self._lock.acquire()
+        # Deep copy to external variable
+        temp_flag = copy.deepcopy(self._arm_trajectory_started)
```

Comment:
Turns out that deepcopy is completely unnecessary here since `self._arm_trajectory_started` is the simple variable type of Bool.  This is something I really should have known before, but I ran this simple test in my python shell:
```
# Assignment
>>> a = 5
>>> b = a
>>> b
5
>>> a = 6
>>> b
5
# Shallow Copy
>>> import copy
>>> b = copy.copy(a)
>>> b
6
>>> a = 7
>>> b
6
# Deep Copy
>>> b = copy.deepcopy(a)
>>> b
7
>>> a = 8
>>> b
7
```
Based on that, any form of copy is unnecessary, and assignment is fine.